pkg/assets/bindata.go

```diff
@@ -150,7 +151,7 @@ func assetsComponentsHostpathProvisionerClusterroleYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "assets/components/hostpath-provisioner/clusterrole.yaml", size: 609, mode: os.FileMode(436), modTime: time.Unix(1654679854, 0)}
+	info := bindataFileInfo{name: "assets/components/hostpath-provisioner/clusterrole.yaml", size: 609, mode: os.FileMode(420), modTime: time.Unix(1654679854, 0)}
```

Comment:
If FileMode and ModTime are common to all the bindataFileInfo calls, should we define them as constants in one place and use consistently?

Reply:
This file is auto-generated by `go-bindata` (see `scripts/bindata.sh`). I'm not sure why the file mode values have all changed. Perhaps that's a difference in a new version of the tool?